### ISSUE TITLE
4.8.8: consider some UI love for Date format preferences

### ISSUE THREAD

**Issue by phiw13** [textpattern/textpattern#1762]
Site > Preferences > Date format (and Archive format) these two perfs now have both a `<select />` widget and a text `<input />` field.

This two sets of input fields (custom date formats) may need some additional UI love: `label` or `aria-label` (or/and `title attribute…).That may help clarify usage as well.

Fine-tuned layout might be a bonus point.

**Comment by philwareham** [comment]
Will investigate

**Comment by philwareham** [comment]
@phiw13 let me know if the above commit improves matters.

**Comment by phiw13** [comment]
I suppose it does (improve things). Not entirely weather “code” is the most appropriate description of those 2 fields as the purpose is to add the ability to customise the date format. But OK.

**Comment by philwareham** [comment]
Yeah agreed, but I didn't want to introduce another text string, and I couldn't think of a suitable one ("Output" didn't sound right either). Open to suggestions.
